template/template.go

```diff
@@ -238,6 +238,20 @@ func NewTemplateExpander(text string, name string, data interface{}, timestamp c
 				}
 				return fmt.Sprintf("%.4g%ss", v, prefix)
 			},
+			"humanizeTimestamp": func(v interface{}) string {
+				switch ts := v.(type) {
```

Comment:
The console templates are primarily for end-users who aren't aware of the complexity of how this is made to work mostly seamlessly under the covers . I'm wary of adding support for something that standard use should never come across, and I'd rather not have to end up having to add automatic type detection to all of the function (including the builtins that come with the templates, which is why everything is switched to float64).